state/prover/proof_op.go

```diff
@@ -0,0 +1,136 @@
+package prover
```

Comment:
Additionally, we can consider switching back to IBC prover, but we also have to verify it for the correctness

cc @renaynay

Reply:
resolved?